src/parse-args.js

```diff
@@ -6,22 +6,24 @@ export default function parseArgs(rawArgs) {
     string: [
       'schema-file',
       'outdir',
-      'schema-bundle-name'
+      'schema-bundle-name',
+      'white-list-type'
     ],
     default: {
       'schema-bundle-name': 'Schema',
       'bundle-only': false
     }
   });
 
-  if (args.help || !(args['schema-file'] || args.outdir)) {
+  if (args.help || !(args['schema-file'] && args.outdir)) {
     return {showHelp: true};
   }
 
   return {
     schemaFile: args['schema-file'],
     outdir: args.outdir,
     schemaBundleName: args['schema-bundle-name'],
-    bundleOnly: args['bundle-only']
+    bundleOnly: args['bundle-only'],
+    whiteListTypes: args['white-list-type'] || false
```

Comment:
/shrug. I feel like an empty array has some misleading semantics? But this is a much better way to be type safe.